building-protected-and-paywall.mdx

```diff
@@ -0,0 +1,969 @@
+---
+title: 'How to build paywall pages with Astro, Stripe, Lucia Auth, and Xata'
```

Comment:
Thinking in form of how a user would search on google, how about having

"Build Authenticated and Paywall Pages with Stripe and Xata"

As the important "services" used in the setup are Stripe and Xata, Astro and Lucia Auth can still be replaced by another setup to get the same thing done with Xata and Stripe.

Does that make sense?

Reply:
That works for me. I was also optimizing for SEO 😆 Lucia Auth get some nice hits with our audience. Let's go with your suggestion.

Reply:
Ah sounds good, let's go with that. Both work.

"Build Authenticated and Paywall Pages with Stripe, Lucia Auth and Xata"